executors/node/plural_rules.js

```diff
@@ -0,0 +1,55 @@
+// Tests Intl Locale for pluralrules.
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale
+
+
+// TODO: Consider if a list of locales can be used.
+module.exports = {
+  testPluralRules: function (json) {
+    const label = json['label'];
+    const locale = json['locale'];
+
+    let return_json = {'label': label};
+
+
+    let test_options = {};
+    let sample;
+    if (json['sample']) {
+      sample = Number(json['sample']);  // Convert to a number
+    }
+    let plural_type;
+    if (json['type']) {
+      plural_type = json['plural_type'];
+      test_options['type'] = plural_type;
+    }
+
+    const supported_locales =
+          Intl.PluralRules.supportedLocalesOf(locale, test_options);
+    console.log("supported: ", supported_locales);
```

Comment:
This could get noisy. The list of supported locales won't change from one invocation to the next, so are we logging the same info for each test case?